server/views/pages/vary/caseload.njk

```diff
@@ -55,7 +55,7 @@
                 id: 'licence-status-' + loop.index,
                 "data-sort-value": offender.licenceStatus | getStatusOrder
             },
-            html: statusBadge({ status: offender.licenceStatus, size: 'small' }, statusConfig) if offender.licenceStatus !== 'ACTIVE'
```

Comment:
Introduced the Active status.
This is for initial licences which have been approved and made active (by the system on CRD or manually by our team) and also licence variations which have been made active.